torch/distributions/utils.py

```diff
@@ -3,7 +3,7 @@
 import torch
 import torch.nn.functional as F
 from typing import Dict, Any
-
+from torch.overrides import has_torch_function
 
 def broadcast_all(*values):
```

Comment:
Would it be possible to add type annotation here (not sure what is the type of torch.Function, I guess `Callable[torch.Tensor]`)?
```suggestion
def broadcast_all(*values: Union[torch.Tensor, Number]):
```

Reply:
I don't think there is a type annotation for objects that implement __torch_function__. Unless PyTorch has implemented an abstract class for this somewhere, but in this case it's not well documented.
And this suggestion would give the impression that Tensor-Likes wouldn't be allowed.